include/seqan3/core/char_operations/predicate_detail.hpp

```diff
@@ -101,7 +101,7 @@ class char_predicate_base;
  * \details
  *
  * An object of the type must be invocable with a std::integral type and supply a static constexpr `msg` member of type
- * seqan3::small_string.
+ * seqan3::string.
```

Comment:
```suggestion
 * std::string.
```
!

Reply:
you might want to grab for this in your changes

Reply:
I did this alredy. Sorry, I rebased because I thought this can be then merged already.

Reply:
huh, But it is still in the code?

Reply:
the diff shows that it is not in the code anymore. I also don't see it in my checkout 🤷‍♂

Reply:
🤦‍♂ didn't see the actual issue.